Meta/build-root-filesystem.sh

```diff
@@ -55,6 +55,7 @@ chmod 0400 mnt/res/kernel.map
 chmod 0400 mnt/boot/Kernel
 chmod 4750 mnt/bin/su
 chmod 4755 mnt/bin/passwd
+chmod 4755 mnt/bin/pls
```

Comment:
Not a big deal, but read permissions for everyone isn't necessary - only execute:

```suggestion
chmod 4751 mnt/bin/pls
```

In the future, it may be a good idea to also add a `sudo` or `pls` group. `/bin/pls` could then have `4750` permissions and `root:pls` ownership to prevent execution by users not in the correct group.

Reply:
Alright I'll get that changed. I'll keep iterating on this once it's in